src/files-and-uploads/data/utils.js

```diff
@@ -46,6 +51,23 @@ export const getSrc = ({ thumbnail, wrapperType, externalUrl }) => {
   }
 };
 
+export const getFileSizeToClosestByte = (fileSize, numberOfDivides = 0) => {
```

Comment:
We need a separate unit test for this to know that is works correctly and will not break in the future.